### PR TITLE
redefine logging levels

### PR DESCRIPTION
**Pull request by pciavald**
Following #344 logging levels (info, debug, error...) have been redefined.

Instead of using `-printtoconsole` and `-debug`, you can now simply use `-v` and `-vv`

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -9,11 +9,29 @@ const platform      = require('os').platform();
...
+log.transports.file.file     = log.transports.file
    .findLogPath(log.transports.file.appName)
    .replace('log.log', 'particl.log');
+
+log.transports.console.level = 'info';
+log.transports.file.level = 'debug';
+
+if (process.argv.includes('-v')) {
+  log.transports.console.level = 'debug'; /* most messages */
+  process.argv.push('-printtoconsole');   /* daemon output */
```
Comment (kewde) [inline main.js]: Can we maybe put the logging to console defined by another argument?
I'd like to be able to get general debug messages (Node backend), without the core logs.

**Comment by kewde** [comment]
Can be merged after fixing the conflicts.

File: main.js (hunk 1 above)
Comment (pciavald) [reply main.js]: Absolutely, thanks for noticing. Here's what I propose : 
-v : gui[debug]
-vv : gui[debug] + daemon[normal]
-vvv : gui[silly] + daemon[debug]

Comment (pciavald) [reply main.js]: @kewde

**Pull request closed without merging**